src/pages/SearchPage.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Link } from '@reach/router'
+import PetCard from './../components/PetCard'
+import LabeledSelect from './../components/common/LabeledSelect'
+import { petsDB, typeOpt, dogBreedsOpt, ageOpt, sizeOpt, genderOpt } from './../petsDB'
+
+const SearchPage = (props) => {
+    const [type, setType] = useState("Animal");
+    const [breeds, setBreeds] = useState([]);
+    const [age, setAge] = useState([]);
+    const [size, setSize] = useState([]);
+    const [gender, setGender] = useState("");
+
+    return (
+        <Wrapper>
+            <SidePanel>
+                <Selectors>
+                    <LabeledSelect 
+                        value={type}
+                        label="TYPE"
+                        options={typeOpt}
+                        onChangeHandler={setType}
+                    />
+                    <LabeledSelect
+                        isMulti={true}
+                        label="BREEDS"
+                        options={dogBreedsOpt}
+                        value={breeds}
+                        onChangeHandler={setBreeds}
+                    />
+                    <LabeledSelect
+                        isMulti={true}
+                        label="AGE"
+                        options={ageOpt}
+                        value={age}
+                        onChangeHandler={setAge}
+                    />
+                    <LabeledSelect
+                        isMulti={true}
+                        label="SIZE"
+                        options={sizeOpt}
+                        value={size}
+                        onChangeHandler={setSize}
+                    />
+                    <LabeledSelect
+                        value={gender}
+                        label="GENDER"
+                        options={genderOpt}
+                        onChangeHandler={setGender}
+                    />
+                </Selectors>
+            </SidePanel>
+            <ListingContainer>
+                <Title>
+                    Available {type}s for Adoption:
+                </Title>
+                <ListingWrapper>
+                    {petsDB.map((pet) =>
```

Comment:
```suggestion
                    {petsDB.map(({name, id, imgs}) =>
```

Reply:
fixed, also in PetListingPreview Component